src/app/forms/services/dynamic-form.types.ts

```diff
@@ -79,7 +84,7 @@ export interface FormNode {
   delimited?: { regex?: string; separator: string };
   value?: any;
   path?: string;
-  options?: FormNodeOption<string | number | boolean | null>[] | FormNodeCombinationOptions;
+  options?: FormNodeOption<string | number | boolean | null>[] | FormNodeCombinationOptions | FormNodeSelectMap;
```

Comment:
Instead of introducing a new interface, is it not possible to use the existing `FormNodeOption<string | number | boolean | null>[] | FormNodeCombinationOptions` union type on `options` as your return type in your new `getOptionsFromObject` function, as we should want to match what options is expecting instead of extending it's type.

e.g.
`getOptionsFromObject(object: object): FormNodeOption<string | number | boolean | null>[] | FormNodeCombinationOptions`

Reply:
We just did this together

Reply:
What do you think of the name? I think it should have something like getSelectOptionsFromObject?

Reply:
Nathan helped resolve by terrible interface ^_^;

Reply:
Also for future reference the interface was not needed as it was the same thing as `Record<string,any>` 😉